test/regression/base/generic/redundant_min_unsat_core2_full.smt2.expected.out

```diff
@@ -0,0 +1,2 @@
+unsat
+( (and b1 b2) (xor b2 b1) )
```

Comment:
There seems to be some problem here, the arguments of `xor` are swapped.
Maybe some bug in the printing?

Reply:
I noticed that but did not care too much - I thought it is related to some preprocessing.
Nonetheless, I think this should be resolved in a separate issue.
-> #776

Reply:
OK!